www/new-prebid-limes.uk/default.html

```diff
@@ -0,0 +1,378 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="icon" type="image/svg+xml" href="noun_Swan_3263882.svg">
+  <title>New Prebid Limes | SWAN Demo</title>
+  <link href="bootstrap.min.css" rel="stylesheet">
+  <link href="blog.css" rel="stylesheet">
+  <link href="advert.css" rel="stylesheet">
+
+  <script>
+    var FAILSAFE_TIMEOUT = 10000;
+
+    window.swan_offers = {
```

Comment:
swan offers will have to be created by the pub, either when the page is fetched or in a separate call